packages/x-charts/src/internals/plugins/featurePlugins/useChartPolarAxis/computeAxisValue.ts

```diff
@@ -0,0 +1,211 @@
+import { scaleBand, scalePoint, scaleTime } from '@mui/x-charts-vendor/d3-scale';
+import { AxisConfig, ScaleName } from '../../../../models';
+import {
+  ChartsAxisProps,
+  ChartsYAxisProps,
+  isBandScaleConfig,
+  isPointScaleConfig,
+  ChartsRotationAxisProps,
+  ChartsRadiusAxisProps,
+} from '../../../../models/axis';
+import { ChartSeriesType, PolarChartSeriesType } from '../../../../models/seriesType/config';
+import { getColorScale, getOrdinalColorScale } from '../../../colorScale';
+import { getTickNumber } from '../../../../hooks/useTicks';
+import { getScale } from '../../../getScale';
+import { getAxisExtremum } from './getAxisExtremum';
+import type { ChartDrawingArea } from '../../../../hooks';
+import { ChartSeriesConfig } from '../../models/seriesConfig';
+
+import { ProcessedSeries } from '../../corePlugins/useChartSeries/useChartSeries.types';
+
+import { DefaultizedAxisConfig } from '../useChartCartesianAxis/useChartCartesianAxis.types';
+
+const degreeToRad = (value?: number, defaultRad?: number) => {
+  if (value === undefined) {
+    return defaultRad!;
+  }
+  return (Math.PI * value) / 180;
+};
+
+type RotationConfig = AxisConfig<ScaleName, any, ChartsRotationAxisProps>;
+
+function getRange(
+  drawingArea: ChartDrawingArea,
+  axisDirection: 'rotation' | 'radius',
+  axis:
+    | AxisConfig<'linear', any, ChartsRadiusAxisProps>
+    | AxisConfig<ScaleName, any, ChartsRotationAxisProps>,
+) {
+  if (axisDirection === 'rotation') {
+    if (axis.scaleType === 'point') {
+      const angles = [
+        degreeToRad((axis as RotationConfig).startAngle, 0),
+        degreeToRad((axis as RotationConfig).endAngle, 2 * Math.PI),
+      ];
+      const diff = angles[1] - angles[0];
+      if (diff > Math.PI * 2 - 0.1) {
```

Comment:
Maybe I'm missing some context here. What is the reason for subtracting specifically by 0.1?

Reply:
To take into account the risk of computation errors that could lead to `diff === 2*Math.PI` being false for less that 0.1

We could move that to 0.001. But I assume polar axes are either displayed on 360° in which case we need to avoid the overlap of first and last point. Or 180° in which case we have nothing to do